### PR TITLE
feat: bold help shortcut

### PR DESCRIPTION
**Pull request by btea**

### CODE REVIEW

**Comment by antfu** [comment]
Can you provide more context? I found it looks fine to me
image

**Comment by btea** [comment]
It looks like this on my side.  My system is windows 10,  and the vscode version is `1.63.2`.

image

**Comment by btea** [comment]
I found that press `enter` also to rerun all tests. Maybe should add it like this `press a/enter to return all tests`? @antfu

File: packages/vitest/src/node/stdin.ts [hunk 1]
```diff
@@ -12,8 +12,8 @@ const keys = [
 export function printShortcutsHelp() {
   process.stdout.write(
     `
-${c.bold('Watch Usage')}
-${keys.map(i => c.dim('  press ') + c.reset(i[0]) + c.dim(` to ${i[1]}`)).join('\n')}
+${c.bold('  Watch Usage')}
+${keys.map(i => c.dim('  press ') + c.bold(c.reset(i[0])) + c.dim(` to ${i[1]}`)).join('\n')}
```
Comment (btea) [inline packages/vitest/src/node/stdin.ts:16]: `c.bold(c.reset(i[0]))` doesn't seem to bold words, maybe we should use `c.reset(c.bold(i[0]))`.  

I did the following test
image

**Pull request merged**